dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -6,7 +6,7 @@
 #
 # Please locate new tests for RegistrationsController in files for individual
 # routes under
-#   test/controllers/registrations_controller/*_test.rb
+#   test/integration/registration/*_test.rb
```

Comment:
should this be 
```suggestion
#   test/integration/omniauth/*_test.rb
```

Reply:
Both, I think.

![image](https://user-images.githubusercontent.com/1615761/86627782-b19e9a00-bf7d-11ea-89ae-b6bd7cd76c6f.png)